kubeflow/core/kubeform_spawner.py

```diff
@@ -237,6 +237,9 @@ def _expand_user_properties(self, template):
 else:
     c.JupyterHub.authenticator_class = 'dummyauthenticator.DummyAuthenticator'
 
+if os.environ.get('DEFAULT_JUPYTERLAB') == 'true':
```

Comment:
Can you ignore case please so that "TRUE" will also work?

Reply:
Done